cmd/editaccount.go

```diff
@@ -93,6 +93,7 @@ func createEditAccount() *cobra.Command {
 	cmd.Flags().VarP(&params.streams, "js-streams", "", "Jetstream: set maximum streams for the account (-1 is unlimited)")
 	params.consumer = -1
 	cmd.Flags().VarP(&params.consumer, "js-consumer", "", "Jetstream: set maximum consumer for the account (-1 is unlimited)")
+	cmd.Flags().VarP(&params.haResources, "js-ha-resources", "", "Jetstream: set maximum high availability resources, such as replicated streams and durable consumer, for the account (-1 unlimited / 0 ha resources disabled)")
```

Comment:
by default we write `0`? If not set what is the behaviour.

Reply:
current default is 0 as is the case with other js limits.

```
~/test/creds-issue  nsc add account test2
[ OK ] generated and stored account key "ABJ2ZAS62SQH3A6KREDFRJS4O4BTVQVVHN552HCQYKEQPOIRPXYQHXXF"
[ OK ] added account "test2"
 ~/test/creds-issue  nsc edit account --name test2  --js-disk-storage 1
[ OK ] changed max disk storage to 1
[ OK ] edited account "test2"
 ~/test/creds-issue  nsc describe account test2
+--------------------------------------------------------------------------------------------+
|                                      Account Details                                       |
+---------------------------------+----------------------------------------------------------+
| Name                            | test2                                                    |
| Account ID                      | ABJ2ZAS62SQH3A6KREDFRJS4O4BTVQVVHN552HCQYKEQPOIRPXYQHXXF |
| Issuer ID                       | OBNTQREHEIPRETMAVPVQUCIGERKGZB3ERAV5S5GM3IOEQNHREBJOUARF |
| Issued                          | 2022-02-10 00:31:53 UTC                                  |
| Expires                         |                                                          |
+---------------------------------+----------------------------------------------------------+
| Max Connections                 | Unlimited                                                |
| Max Leaf Node Connections       | Unlimited                                                |
| Max Data                        | Unlimited                                                |
| Max Exports                     | Unlimited                                                |
| Max Imports                     | Unlimited                                                |
| Max Msg Payload                 | Unlimited                                                |
| Max Subscriptions               | Unlimited                                                |
| Exports Allows Wildcards        | True                                                     |
| Response Permissions            | Not Set                                                  |
+---------------------------------+----------------------------------------------------------+
| Jetstream                       | Enabled                                                  |
| Max Disk Storage                | 1 B                                                      |
| Max Mem Storage                 | Disabled                                                 |
| Max Streams                     | 0                                                        |
| Max Consumer                    | 0                                                        |
| Max High Availability Resources | 0                                                        |
+---------------------------------+----------------------------------------------------------+
| Imports                         | None                                                     |
| Exports                         | None                                                     |
+---------------------------------+----------------------------------------------------------+
 ~/test/creds-issue 
```

Reply:
this is basically the init value of the struct.